frontend/src/App.vue

```diff
@@ -1,3 +1,21 @@
 <template>
-  <router-view></router-view>
-</template>
\ No newline at end of file
+  <v-layout class="bg-blue-lighten-1">
+    <v-main class="app">
+      <router-view />
+    </v-main>
+  </v-layout>
+</template>
+
+<style>
+  * {
```

Comment:
This is CSS reset stuff. Better to create a _reset.css_ file and import it in _main.ts_
you can use this one for example [https://gist.github.com/Asjas/4b0736108d56197fce0ec9068145b421](https://gist.github.com/Asjas/4b0736108d56197fce0ec9068145b421)

Reply:
Deleted styles for "*", cause vuetify file with styles already has these styles